config/pom_template.xml

```diff
@@ -17,6 +17,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
+#{version_properties}
```

Comment:
@dylanShark can we make it so that this is used by the dependencymanagement pom generator, but not by the "regular" dynamicpomgen?  For now I think we only want these properties for the dependency management pom (and pom template of course)